src/pager.rs

```diff
@@ -40,16 +40,24 @@ impl PagerKind {
     fn from_bin(bin: &str) -> PagerKind {
         use std::path::Path;
 
-        match Path::new(bin)
-            .file_stem()
-            .map(|s| s.to_string_lossy())
-            .as_deref()
-        {
+        let file_stem = Path::new(bin).file_stem();
+
+        match file_stem.map(|s| s.to_string_lossy()).as_deref() {
             Some("bat") => PagerKind::Bat,
             Some("less") => PagerKind::Less,
             Some("more") => PagerKind::More,
             Some("most") => PagerKind::Most,
-            _ => PagerKind::Unknown,
+            _ => {
+                if env::args_os()
+                    .nth(0)
+                    .map(|arg0| Path::new(&arg0).file_stem() == file_stem)
+                    .unwrap_or(false)
+                {
+                    PagerKind::Bat
```

Comment:
Just an idea, not sure if it works: Maybe we can match on `our_bin_name` earlier and keep the ` _ => PagerKind::Unknown,` line unchanged?

Reply:
Sounds like a good idea.